bindings/ruby/lib/typelib/gccxml.rb

```diff
@@ -695,6 +647,7 @@ def self.parse_cxx_documentation_before(lines, line)
         def load(required_files, xml)
             @registry = Typelib::Registry.new
             Typelib::Registry.add_standard_cxx_types(registry)
+            registry.alias('/std/basic_string</char>', '/std/string')
```

Comment:
Shouldn't this be moved into `add_standard_cxx_types`? It left  my memory why this manual alias is not needed for the gccxml-case...